storage/client/src/main/java/com/azure/storage/blob/BlobAsyncRawClient.java

```diff
@@ -256,8 +256,9 @@
      */
     public Mono<DownloadAsyncResponse> download(BlobRange range, BlobAccessConditions accessConditions,
                                              boolean rangeGetContentMD5, Context context) {
-        Boolean getMD5 = rangeGetContentMD5 ? rangeGetContentMD5 : null;
         range = range == null ? new BlobRange(0) : range;
```

Comment:
If you don't specify a range, it'll default to downloading the whole blob. I think the reason we wanted to get an equivalent result without a null object here is because it makes the logic below easier since we check range values.